tasks.py

```diff
@@ -1125,6 +1126,10 @@ def service(
         The main reason this is a separate task is to make it easy to start services
         when running tests locally.
     """
+
+    class EnvVars(BaseSettings):
+        ci: Optional[bool] = None
+
```

Comment:
```python
    class EnvVars(BaseSettings):
        ci: Optional[bool] = None

        env_vars = EnvVars()
```
You should be able to instantiate once rather than re-parsing for every service.
My mistake in the initial suggestion.

Also I surprised you had todo `EnvVars.update_forward_refs()`, but you could also do that once. :shrug: